modules/core/core-api/src/main/java/com/enonic/xp/content/ModifiableSite.java

```diff
@@ -0,0 +1,24 @@
+package com.enonic.xp.content;
+
+import com.enonic.xp.annotation.PublicApi;
+import com.enonic.xp.site.Site;
+import com.enonic.xp.site.SiteConfigs;
+
+@PublicApi
+public class ModifiableSite
```

Comment:
we might need to reconsider this. site configs exist for root content, too. So maybe it should be part of any content?